data/gui/window/game_version.cfg

```diff
@@ -331,328 +380,459 @@
+																						[/column]
+																						[column]
+																							grow_factor = 0
+																							horizontal_alignment = left
+																							border = "left"
+																							border_size = 5
+																							
+																							[label]
+																								label = _ "<span foreground='yellow'><u>https://wiki.wesnoth.org</u></span>"
+																								use_markup = yes
+																								link_aware = yes
+																							[/label]
+																						[/column]
+																					[/row]
+																				[/grid]
+																			[/column]
+																		[/row]
+																	[/grid]
+																[/column]
+															[/row]
+														[/layer]
+														
+														[layer]
+															{_GUI_VERINFO_TAB_PAGE_GAME_PATHS}
+														[/layer]
+														
+														[layer]
+															{_GUI_VERINFO_TAB_PAGE_BUILD_INFO}
+														[/layer]
+														
+														[layer]
+															{_GUI_VERINFO_TAB_PAGE_GAME_FEATURES}
+														[/layer]
+														
+													[/stacked_widget]
+													
+												[/column]
+												
+											[/row]
+											
+										[/grid]
+										
+									[/column]
+									
+								[/row]
+								
+							[/grid]
+							
+						[/column]
+						
+					[/row]
+					
+					[row]
+						[column]
+							[spacer]
+								height = 30
+							[/spacer]
+						[/column]
+					[/row]
+					
+					[row]
+						grow_factor = 0
+						
+						[column]
+							horizontal_grow = true
+							
+							[grid]
+								
+								[row]
+									[column]
+										horizontal_alignment = left
+										[grid]
+											[row]
+												[column]
+													grow_factor = 1
+													horizontal_alignment = left
+													border = "all"
+													border_size = 5
+													
+													[button]
+														id = "credits"
+														definition = "default"
+														
+														label = _ "Credits"
+													[/button]
+													
+												[/column]
+												
+												[column]
+													grow_factor = 1
+													horizontal_alignment = left
+													border = "all"
+													border_size = 5
+													
+													[button]
+														id = "license"
+														definition = "default"
+														
+														label = _ "License"
+													[/button]
+													
+												[/column]
+												
+												[column]
+													[spacer]
+														width = 30
+													[/spacer]
+												[/column]
+												
+												[column]
+													grow_factor = 1
+													horizontal_alignment = left
+													border = "all"
+													border_size = 5
+													
+													[button]
+														id = "issue"
+														definition = "default"
+														
+														label = _ "Report an issue"
```

Comment:
The label for this should be "Report Issue":

1. Button labels should be Title Case ("Report an Issue")
2. Avoid skippable grammatical elements in widget labels since they only help make wordy labels even wordier, especially when translated, and for buttons in particular, long labels are a big no-no

Reply:
Updated.